### PR TITLE
sqrtm #6549

### PR DESCRIPTION
**Pull request by amrtsg**
sqrtm #6549

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py [hunk 1]
```diff
@@ -508,3 +508,27 @@ def test_tensorflow_l2_normalize(
...
     )
+
+
+# sqrtm
+@handle_cmd_line_args
+@given(
+    dtype_and_input=_get_dtype_and_matrix(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.linalg.sqrtm"
+    ),
```
Comment (simonetgordon) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:519]: Hi there!

Thanks for the contribution.

First thing: yout need to assign strategies for all inputs to `test_tensorflow_sqrtm` in the @given decorator.

Comment (amrtsg) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:519]: Not sure I understand exactly what you mean, could you explain further? Should I check for all valid datatypes and inputs?

Comment (simonetgordon) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:519]: You need to define each of the arguments for the test function in the given decorate, define the hypothesis strategy. Please have a read of of the hypothesis test writing [section](https://lets-unify.ai/ivy/deep_dive/ivy_tests.html) of the docs for more details :)

Comment (simonetgordon) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:519]: > You need to define each of the arguments for the test function in the given decorate, define the hypothesis strategy. Please have a read of of the hypothesis test writing [section](https://lets-unify.ai/ivy/deep_dive/ivy_tests.html) of the docs for more details :)

My apologies, I was incorrect about this. These arguments are handled by the `@handle_cmd_line_args` decorator.

File: ivy/functional/frontends/tensorflow/linalg.py [hunk 2]
```diff
@@ -111,3 +111,8 @@ def l2_normalize(x, axis=None, epsilon=1e-12, name=None):
     square_sum = ivy.sum(ivy.square(x), axis=axis, keepdims=True)
     x_inv_norm = ivy.reciprocal(ivy.sqrt(ivy.maximum(square_sum, epsilon)))
     return ivy.multiply(x, x_inv_norm)
+
+
+@to_ivy_arrays_and_back
+def sqrtm(input, name=None):
+    return ivy.sqrtm(input)
```
Comment (simonetgordon) [inline ivy/functional/frontends/tensorflow/linalg.py:118]: The reason for the failure is that `ivy.sqrtm` has not been implemented.

Please the [frontend api contributor guide](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis) for options on how to proceed. Basically, you can try and implement sqrtm in the frontend as a composition of ivy functions, or if that's too tricky, you can raise an issue requesting that `sqrtm` be implemented in the ivy backends and work on another function.

**Review by simonetgordon — CHANGES_REQUESTED** [review]

**Comment by simonetgordon** [comment]
@amrtsg Hi there, are you still working on this PR or can I close it?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**